config/virtual/config.go

```diff
@@ -35,6 +36,12 @@ func Configure(p *config.Provider) {
 				Extractor: rconfig.APISPackagePath + "/rconfig.ExtractResourceName()",
 			},
 		}
+		r.ExternalName = config.NameAsIdentifier
+		r.ExternalName.GetExternalNameFn = common.GetNameFromFullyQualifiedID
+		// /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/mygroup1/providers/Microsoft.Network/virtualNetworks/myvnet1
+		r.ExternalName.GetIDFn = common.GetFullyQualifiedIDFn("Microsoft.Network",
+			"virtualNetworks", "name",
+		)
 	})
 
 	p.AddResourceConfigurator("azurerm_virtual_network_gateway", func(r *config.Resource) {
```

Comment:
These resources are in `network` group but config file still resides in `virtual` folder, mind making sure all lives in its own folder so that it's easier to browse through?